helm/app/templates/routes/secureRoute.yaml

```diff
@@ -32,3 +32,31 @@ spec:
     name: {{ template "ccbc.fullname" . }}
     weight: 100
   wildcardPolicy: None
+
+{{- if secureRoute.wwwRoute.enable }}
+---
+# The www route isn't managed by certbot to avoid duplicate certificate requests
```

Comment:
tech debt: bcgov certbot should not attempt to fetch any cert matching `/^www\./`, would love to see this risk tracked in backlog and brought to PO. Per new ch12 policy we should be sending fix for this upstream if at all possible.